packages/sveltekit/src/client/sdk.ts

```diff
@@ -41,15 +41,13 @@ export function init(options: BrowserOptions): Client | undefined {
 }
 
 function getDefaultIntegrations(options: BrowserOptions): Integration[] | undefined {
-  // This evaluates to true unless __SENTRY_TRACING__ is text-replaced with "false", in which case everything inside
-  // will get treeshaken away
+  // This evaluates to true unless __SENTRY_TRACING__ is text-replaced with "false",
+  // in which case everything inside will get tree-shaken away
   if (typeof __SENTRY_TRACING__ === 'undefined' || __SENTRY_TRACING__) {
-    if (hasTracingEnabled(options)) {
-      return [...getDefaultSvelteIntegrations(options), svelteKitBrowserTracingIntegration()];
-    }
+    return [...getDefaultSvelteIntegrations(options), svelteKitBrowserTracingIntegration()];
+  } else {
+    return getDefaultSvelteIntegrations(options);
   }
```

Comment:
l: we don't need to throw the last `return` statement into an `else` block but we can just return directly. Not a problem but it shaves off a couple of bytes and reduces blocks :)